app/views/posts/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>List of all posts by user </h1>
\ No newline at end of file
+
+<div>
+
+  <h3 class="border ps-2 w-25 m-2 p-4 border-dark"> <%= @user.name %> </h3> <br>
+     
+      <% @user.recent_posts.each do |post| %>
+
+       <div>
+       <div class="border h-50 w-50 p-2 m-2 border-dark">
+        <%= post.title %> <br>
+        <%= post.text %>  
+        </div> 
+        <div class="border h-50 w-50 p-2 m-2 border-primary">
+            <% post.recent_comments.each do |comment| %>
+              <p><%= comment.author.name %>
+              <%= comment.text %></p>
+              <% end %>
+        </div> <br>
+       </div>
+      <% end %>
+<div>
```

Comment:
- [ ] Again, please consider following the wireframe and adding the remaining user details (photo, post_count) and comments and likes counter: 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/222304721-627a8670-300a-409d-a64d-19201fe9ca35.png) | ![image](https://user-images.githubusercontent.com/77069011/222305013-905aa3e4-933a-473a-94c4-2998a4c64eac.png) |